app/views/user_mailer/abuse_report.text.erb

```diff
@@ -17,10 +17,10 @@
 <%= to_plain_text(@url) %>
 
 <%= work_metadata_label(t(".copy.summary")) %>
-<%= to_plain_text(raw @summary) %>
+<%= to_plain_text(raw(strip_images(@summary))) %>
```

Comment:
Doesn't `to_plain_text` remove the image?

https://github.com/otwcode/otwarchive/blob/acebcaddee50ce6d6d1503f22666ea47b6891f91/app/helpers/mailer_helper.rb#L77-L93

It looks like it uses `strip_tags`, which removes the `img` tag, including the source URL, if I do something like

```
<% text = "<p>idk an image <img src=\"https://ao3.org/images/logo.png\"> yay</p>" %>
<%= strip_tags(text) %>
```